tools/screenshots/overview/konnect-dashboard.yaml

```diff
@@ -0,0 +1,7 @@
+- output: ../../app/_assets/images/docs/konnect/konnect-dashboard2.png
```

Comment:
@Guaris Where can you see this one (like the URL of the page it's on, I looked at the main Overview page, but didn't see it there)?

Reply:
Ah, you caught a mistake here. I didn't add the image but I forgot to rename this file back to dashboard2.png